packages/amplify_datastore_plugin_interface/test/testData/Person.dart

```diff
@@ -13,30 +13,21 @@
 * permissions and limitations under the License.
 */
 
-// ignore_for_file: public_member_api_docs
-
-import 'ModelProvider.dart';
 import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
 import 'package:collection/collection.dart';
 import 'package:flutter/foundation.dart';
 
+// ignore_for_file: public_member_api_docs
+
+import 'ModelProvider.dart';
+
 /** This is an auto generated class representing the Person type in your schema. */
 @immutable
-class Person extends Model {
-  static const classType = const _PersonModelType();
-  final String id;
+class Person {
```

Comment:
What's up with this class

Reply:
Oops missed this one, seems the model generated by 

```
type Person {
  name: String!
  propertiesAddresses: [Address]
  contact: Contact!
}
```

which should be with `@model`.